pkg/controllers/networkinfo/vpcnetworkconfig_handler.go

```diff
@@ -110,3 +112,50 @@ func getListSize(s []string) int {
 		return len(s)
 	}
 }
+
+func buildNetworkConfigInfo(vpcConfigCR v1alpha1.VPCNetworkConfiguration) (*commontypes.VPCNetworkConfigInfo, error) {
+	org, project, err := nsxtProjectPathToId(vpcConfigCR.Spec.NSXTProject)
+	if err != nil {
+		log.Error(err, "failed to parse nsx-t project in network config", "Project Path", vpcConfigCR.Spec.NSXTProject)
+		return nil, err
+	}
+
+	ninfo := &commontypes.VPCNetworkConfigInfo{
```

Comment:
The name `VPCNetworkConfigInfo` is a little confusing, is it NetworkConfiguration, or NetworkInfo?

Reply:
Written by @seanpang-vmware